roles/satellite-clone/tasks/main.yml

```diff
@@ -190,17 +169,33 @@
   command: restorecon -R /
   when: restorecon
 
-# Run Satellite installer
-- name: run Satellite 6.1 installer
+# This file tells the candlepin puppet module in the Satellite installer that
+# candlepin is already set up. We remove it so candlepin is set up correctly.
+- name: Remove cpdb_done file
```

Comment:
did this cause an issue in the past? If so we can create an issue and link to this PR too.

Reply:
I don't think it has in the past, but it definitely is causing an issue in 6.3.

Reply:
but, given we're restoring a working backup, shouldn't candlepin actually be properly setup at this point?

Reply:
not exactly, here are the rough steps of clone and what is happening
- restore configs  <---- cpdb_done file is restored from the backup
- install Satellite <----- installer sees cpdb_done file and says "ok I wont set up candlepin then", resulting in errors on seed steps when installing
- restore backups (mongo, postgres, pulp data)
- re-run installer

Reply:
Aah, gotcha. I'd argue there should be an RFE to the installer, to force Candlepin re-init (like we can with certs), but that remove will do for now.